RealmSwift/Object.swift

```diff
@@ -79,77 +79,122 @@ public class Object : RLMObjectBase, Equatable {
-             not intended to be used directly.
+    Get an `Array` of objects of type `className` which have this object as the given property value. This can
+    be used to get the inverse relationship value for `Object` and `List` properties.
+    :param: className The type of object on which the relationship to query is defined.
+    :param: property  The name of the property which defines the relationship.
+    :returns: An `Array` of objects of type `className` which have this object as their value for the `propertyName` property.
+    */
+    public func linkingObjects<T: Object>(type: T.Type, forProperty propertyName: String) -> [T] {
+        return linkingObjectsOfClass(T.className(), forProperty: propertyName) as [T]
+    }
 
-    This initializer is called by the Objective-C accessor creation code, and if it's
-    not overridden in Swift, the inline property initializers don't get called,
-    and we require them for `List<>` properties.
 
-    :param: realm         The realm to which this object belongs.
-    :param: schema        The schema for the object's class.
-    :param: defaultValues Whether the default values for this model should be used.
+    // MARK: Private functions
+
+    // FIXME: None of these functions should be exposed in the public interface.
+
+    /**
+    WARNING: This is an internal initializer not intended for public use.
     */
     public override init(realm: RLMRealm, schema: RLMObjectSchema, defaultValues: Bool) {
         super.init(realm: realm, schema: schema, defaultValues: defaultValues)
     }
 
     /**
-    WARNING: This is an internal initializer for Realm that must be `public`, but is
-             not intended to be used directly.
-
-    This initializer is called by the Objective-C accessor creation code, and if it's
-    not overridden in Swift, the inline property initializers don't get called,
-    and we require them for `List<>` properties.
-
-    :param: realm  The realm to which this object belongs.
-    :param: schema The realm's schema.
+    WARNING: This is an internal initializer not intended for public use.
     */
     public override init(object: AnyObject, schema: RLMSchema) {
         super.init(object: object, schema: schema)
     }
 
     /**
-    WARNING: This is an internal initializer for Realm that must be `public`, but is
-             not intended to be used directly.
-
-    This initializer is called by the Objective-C accessor creation code, and if it's
-    not overridden in Swift, the inline property initializers don't get called,
-    and we require them for `List<>` properties.
-
-    :param: objectSchema The schema for the object's class.
+    WARNING: This is an internal initializer not intended for public use.
     */
     public override init(objectSchema: RLMObjectSchema) {
         super.init(objectSchema: objectSchema)
     }
+
+    // Get RLMArray values when getting array properties
```

Comment:
Use triple slashes for documentation in `valueForKey()` and `setValue(_:forKey:)`.